book/src/SUMMARY.md

```diff
@@ -33,6 +33,7 @@
         - [Data Sources](volume-datasources.md)
             - [Cloning](volume-cloning.md)
             - [Volume Snapshot & Restore](snapshot-restore-feature.md)
+            - [Volume Group Snapshot & Restore](group-snapshot.md)
```

Comment:
should we name this file to `grou-snapshot-restore-feature.md` to keep it similar to `volume snapshot`?